modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/metal/js/components/RuleEditor/RuleEditor.es.js

```diff
@@ -883,6 +883,35 @@ describe(
 							}
 						);
 
+						it(
+							'should add to the deletedField all the fields removed from the FormBuilder',
+							() => {
+								component = new RuleEditor(
+									{
+										...getBaseConfig()
+									}
+								);
+
+								component.refs.firstOperand0.emitFieldEdited(['radio']);
+
+								jest.runAllTimers();
+
+								const formBuilderFields = component.fieldOptions.map(a => a.fieldName);
+
+								component.pages = [];
+
+								jest.runAllTimers();
+
+								const notEquals = formBuilderFields.some(
+									item => {
+										return component.deletedFields.indexOf(item) == -1;
```

Comment:
You don't need brackets, you could use item => component.deletedFields.indexOf(item) == -1;